src/pages/Organization/OrganizationPatients.tsx

```diff
@@ -193,12 +194,17 @@ export default function OrganizationPatients({ id, navOrganizationId }: Props) {
                               </div>
                             )}
                           </div>
-                          <div className="mt-4 pt-4 border-t">
-                            <RecordMeta
-                              className="text-sm text-gray-500"
-                              prefix="Last updated"
-                              time={patient.modified_date}
-                            />
+                          <div className="mt-4 pt-4 border-t border-gray-200">
+                            <div className="text-sm text-gray-500">
+                              {t("last_modified")}{" "}
+                              <TooltipComponent
+                                content={formatDateTime(patient.modified_date)}
+                              >
+                                <span>
+                                  {relativeTime(patient.modified_date)}
```

Comment:
add underline to highlight?

<img width="352" alt="image" src="https://github.com/user-attachments/assets/98f95190-901d-45f6-9d44-abc3631e9249" />